index.html

```diff
@@ -400,6 +400,21 @@ <h3>Timeline</h3>
 
 			<section id="success-criteria">
 				<h2>Success Criteria</h2>
+				
+				<p>The EPUB 3 Working Group’s work is considered a success 
+				if all of the following conditions are met:</p>
+				
+				<ol>
+					<li>There are multiple, independent, interoperable implementations 
+					of EPUB 3.X that are widely used.</li>
+					<li>Existing content is supported. All conforming EPUB 3.2 publications should remain conformant to EPUB 3.X.</li>
+					
+					<li>Changes to the global ecosystem 
+					(including authors, publishers, distributors, retailers, reading systems, and tool makers) 
+					are minimized, and made only after careful consideration 
+					of how they would affect all constituencies.</li>
+					<li>Validation tools such as EPUBCheck fully support EPUB 3.X.</li>
+				</ol>
 				<p>
 					In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of each of feature defined in the specification.
```

Comment:
I guess a much more operative item on how the features are handled may be necessary to dissipate (if possible) the mistrust clearly expressed in https://github.com/w3c/epub-3-wg-charter/issues/8#issuecomment-618106944. Adding something like https://github.com/w3c/epub-3-wg-charter/issues/8#issuecomment-617780145 may help (this is not in contradiction with the proposal in this PR!)